classes/class4/homework-soln/app.js

```diff
@@ -0,0 +1,37 @@
+/* REQUIRES */
+// ...npm
+var path = require('path');
+var express = require('express');
+var exphbs = require('express-handlebars');
+var bodyParser = require('body-parser');
+var cookieParser = require('cookie-parser');
+var logger = require('morgan');
+
+// ...local
+var index = require('./routes/index.js');
+
+
+/* CONFIG APP */
+var app = express();
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+app.engine('handlebars', exphbs({defaultLayout: 'main'}));
+app.set('view engine', 'handlebars');
+
+
+/* ROUTING */
+app.get('/ingredients', index.ingredients);
```

Comment:
We should probably add a "/" route. It can just be some welcome text, but would expose the navbar which is handy if you don't know the below routes.